### ISSUE TITLE
Use styling consistent with python.org

### ISSUE THREAD

**Issue by pradyunsg** [pypi/warehouse#2562]
This is really a styling/branding issue and I imagine it won't block anything. 

<img width="1920" alt="screen shot 2017-11-07 at 1 19 04 pm" src="https://user-images.githubusercontent.com/3275593/32482573-52d4ed7a-c3be-11e7-98d7-a45ba446b1ee.png">
<img width="1920" alt="screen shot 2017-11-07 at 1 19 23 pm" src="https://user-images.githubusercontent.com/3275593/32482578-54b122f8-c3be-11e7-90bc-b063d8a19743.png">

For now, I think it would be nice if Warehouse's theme used the same combination of blue/black as python.org for the header section.

Longer term, maybe the theming of pypi.org, python.org, docs.python.org should be the same but that's really a long shot and a lot of work too.

**Comment by user-e** [comment]
Also, a similar layout with less whitespace would be better.

**Comment by Safihre** [comment]
Agreed. Another point I wanted to add to #2417, making the designs consistent makes complete sense to me.
Now there are kind of 50/50 consistent. Still better than 0 consistency of the current PyPi 😃

**Comment by brainwane** [comment]
For the whitespace/condensing issues, let's talk in #1988 and #2417.

Pradyun, thanks for the feedback! Nicole's taking it into account. (You may have already read Nicole (@nlhkabu)'s [design goals, including her assessment of the legacy UI and its problems](http://whoisnicoleharris.com/2015/12/31/designing-warehouse-an-overview.html).) She's [doing user tests to get more data on what users have trouble with](http://whoisnicoleharris.com/2018/03/13/user-testing-warehouse.html), and during those tests, asking about things like design inconsistency between www.python.org, and usual device/browser/display usage and usual size/resolution. Please do signal-boost that request for users to participate in the user tests so she can get more data!

As you said, this is a branding/styling issue and does not need to block the rollout of Warehouse and the shutdown of Legacy PyPI.

**Comment by pradyunsg** [comment]
> Pradyun, thanks for the feedback!

You're welcome! ^>^

Another thing that comes to mind looking at this right now is that python.org links to pypi.python.org from the top bar (the black one). Maybe Warehouse should have it too, for consistency? Just a thought.

See also: https://github.com/python/python-docs-theme/issues/10 

> Nicole's taking it into account.

Awesome! Glad to hear that!

**Comment by brainwane** [comment]
@pradyunsg Once we do the redirect/switch, links to pypi.python.org will redirect to Warehouse, and we'll ask the people who run the python.org website to link directly to the new PyPI. Thanks!

**Comment by pradyunsg** [comment]
Umm... I think you misunderstand... 

> I'm suggesting of using the top-bar (the one with links to PyPI, Docs, Jobs and all) like Ubuntu does (see top of https://www.ubuntu.com, https://www.askubuntu.com and the likes).

**Comment by brainwane** [comment]
Ah so what you meant was that Warehouse should also have *a black navbar at the top of the page*, like python.org currently does. Thanks for clarifying!

**Comment by brainwane** [comment]
A user [said](https://mail.python.org/pipermail/python-list/2018-March/732182.html):

> The back ground blue on the pypi page is the highlight blue on the python.org page, they should change the color to match to background python.org color.

**Comment by pradyunsg** [comment]
Nudge to check what the feelings are on this.

**Comment by miketheman** [comment]
Closing - PyPI has its own branding, and is related-but-not-the-same to python.org.
It's also undergone multiple refreshes - the most recent in #19951 which has updated some of the theme.
